backend/src/db/migrations/20200312140328-bulk_upload_to_s3.js

```diff
@@ -32,39 +32,38 @@ export async function up(next) {
   try {
     // Implement your migration here.
     const { records } = await transaction.run('MATCH (image:Image) RETURN image.url as url')
-    let urls = records.map(r => r.get('url'))
-    urls = urls.filter(url => url.startsWith('/uploads'))
+    let urls = records.map((r) => r.get('url'))
+    urls = urls.filter((url) => url.startsWith('/uploads'))
     const locations = await Promise.all(
       urls
-        .map(url => {
+        .map((url) => {
           return async () => {
             const { pathname } = new URL(url, 'http://example.org')
             const fileLocation = path.join(__dirname, `../../../public/${pathname}`)
             const s3Location = `original${pathname}`
-            const mimeType = mime.lookup(fileLocation)
+            if (existsSync(fileLocation)) {
+              const mimeType = mime.lookup(fileLocation)
+              const params = {
+                Bucket,
+                Key: s3Location,
+                ACL: 'public-read',
+                ContentType: mimeType || 'image/jpeg',
```

Comment:
`mimeType` or whatever :laughing:

Reply:
yeah, `ContentType` is not required, so maybe it doesn't make sense to have it at all?